ui/apps/ui/src/app/pages/trainings-page/training-detail-page.component.html

```diff
@@ -26,84 +26,99 @@
               </div>
               </a>
               <div class="button-off" i18n>
-                OPEN ACCESS
+                {{ myTraining.best_access_right }}
               </div>
             </div>
           </div>
-
-<div style="position: relative; bottom: -117px;">
-  <div class="arrows" style="position: relative; border-top: 1px solid #D8D8D8;">
-    <p i18n class="urls-container" style="padding-left: 64px; height: 41px; margin: 0; display: flex; align-items: center;">
-      <a href="{{myTraining.url}}"><span> -> </span> Webpage </a>
-      <!-- <a><span> -> </span> Helpdesk </a>
-      <a><span> -> </span> Manual </a> -->
-    </p>
-  </div>
-  <div class="footer">
-    <p i18n>ABOUT</p>
-  </div>
-</div>
-
-
-
         </div>
         <div class="details">
-
-
-          <div class="column-left" style="padding-top: 24px; padding-right: 24px;">
-
+          <div class="column-left">
             <p>{{ myTraining.description }}</p>
-            <h2>Learning Outcome(s)</h2>
-            <ng-container *ngFor="let learn of myTraining.learning_outcomes">
-              <p>{{ learn }}</p>
-            </ng-container>
-
-            <div style="display: flex;">
-              <div class="col-4">
-                <p class="section" i18n>Authors<br>
-                  <ng-container *ngFor="let author of myTraining.author_names">
-                  <span class="section-description">{{ author }}</span><br>
-                  </ng-container>
-                </p>
-                <p class="section" i18n>Learning Resource Type<br>
-                  <ng-container *ngFor="let resource of myTraining.resource_type">
-                    <span class="section-description">{{ resource }}</span><br>
-                  </ng-container>
+            <div class="details-box">
+              <div class="details-box-container">
+                <div class="svg-box">
+                  <div class="category-icon">
+                    <span class="helper"></span>
+                    <img [src]="'./assets/light.svg'" alt="">
+                  </div>
+                </div>
+                <p class="details-header" i18n>Basic Details</p>
+              </div>
+              <div class="column-container">
+                <div class="col-4">
+                  <p class="section" i18n>Version Date<br><span class="section-description">{{ myTraining.publication_date | transformDatePipe }}</span></p>
+                  <p class="section" i18n>License<br><span class="section-description">{{ myTraining.license }}</span></p>
+                  <p class="section" i18n>Resource URL type<br><span class="section-description">{{ myTraining?.url_type ?? '-' | filterPipe: 'url_type'  }}</span></p>
+                </div>
+                <div class="col-4">
+                  <p class="section" i18n>Authors<br>
+                    <ng-container *ngFor="let author of myTraining.author_names">
+                      <span class="section-description">{{ author}}, </span>
```

Comment:
@agpul @Marcelinna @aleksandrahojszyk 
I wonder if a comma at he end is necessary here?

Examples:
Author, 
Author, author,

Shouldnt be just?
Author
Author, author

Reply:
done